pandas/tests/window/test_moments.py

```diff
@@ -155,9 +155,41 @@ def test_cmov_window_frame(self):
         rs = DataFrame(vals).rolling(5, win_type="boxcar", center=True).mean()
         tm.assert_frame_equal(DataFrame(xp), rs)
 
-        # invalid method
-        with pytest.raises(AttributeError):
-            (DataFrame(vals).rolling(5, win_type="boxcar", center=True).std())
+        # std
```

Comment:
can you move these to a new test (may also need to rename the original to _mean)

Reply:
I parameterized the test instead. Is that alright?